<launch>
  <param name="use_sim_time" type="bool" value="true" />    
  <arg name="config_file" default="$(find render_kinect)/config/default_setup/robot_sim_config.yaml" />
  <arg name="camera" default="XTION" />
  <arg name="depth_frame" default="/$(arg camera)_IR" />
  <arg name="calibration_url"  default="$(find state_filtering)/config/xtion_calib.yaml" />
  <arg name="background"  default="true" />
  <param name="robot_description" command="$(find xacro)/xacro.py '$(find arm_robot_model)/models/DarpaRobot.urdf.xacro'" />
  <node pkg="render_kinect" name="generate_robot_dataset" type="generate_robot_dataset" respawn="false" output="screen">
    <param name="depth_frame_id"  type="string"  value="$(arg depth_frame)" />
    <param name="calib_url"  type="string"  value="$(arg calibration_url)" />
    <param name="background"  type="bool"  value="$(arg background)" />
    <rosparam command="load" file="$(arg config_file)" />
    <param name="config_file"  type="string"  value="$(arg config_file)" />    
    <param name="robot_description_package_path" type="string" value="$(find arm_robot_model)" />
  </node>
  
</launch>
